testsuite/tests/ui/oas/test_api_key_autocomplete.py

```diff
@@ -42,7 +42,7 @@ def api_doc_page(login, navigator, ui_active_doc, request, custom_admin_login):
     """Custom doc page for developer portal"""
     view = navigator.navigate(CMSNewPageView)
     page_name = blame(request, "CustomDocumentation")
-    page_path = blame(request, "apidocs")
```

Comment:
any reason why `blame(request, "/path")` is not used? https://github.com/3scale-qe/3scale-tests/blob/ad13247b54213999e54580466c6cb67ef802e0fe/testsuite/tests/ui/devel/test_devel_sections.py#L38

Reply:
That solution come to my mind, but yours seems better.